src/XamlStyler.UnitTests/TestConfigurations/SerializedDefault.json

```diff
@@ -12,5 +12,6 @@
       "PageSource, PageIndex, Offset, Color, TargetName, Property, Value, StartPoint, EndPoint",
       "mc:Ignorable, d:IsDataSource, d:LayoutOverrides, d:IsStaticText",
       "Storyboard.*, From, To, Duration"
-  ]
+  ],
+  "IgnoreDesignTimeReferencePrefix": false
```

Comment:
Our documentation is not great in these test files -- for this one, we can leave the new option out.

Reply:
**Corrected**